Makefile.core.mk

```diff
@@ -357,6 +357,10 @@ gen-check: check-no-modify gen check-clean-repo
 sync-configs-from-istiod:
 	cp manifests/charts/istio-control/istio-discovery/files/injection-template.yaml manifests/charts/istiod-remote/files/
 	cp manifests/charts/istio-control/istio-discovery/templates/istiod-injector-configmap.yaml manifests/charts/istiod-remote/templates/
+	cp manifests/charts/istio-control/istio-discovery/templates/configmap.yaml manifests/charts/istiod-remote/templates/
+	cp manifests/charts/istio-control/istio-discovery/templates/configmap-jwks.yaml manifests/charts/istiod-remote/templates/
```

Comment:
this is not needed? its used to be mounted on the istiod pod which isn't present

Reply:
this is needed for gateway

Reply:
seems this is used when jwksResolverExtraRootCA is set.

Reply:
I don't think thats true @irisdingbj . Can you provide some details?

Reply:
I might misunderstand  what you refer to. I mean `cp manifests/charts/istio-control/istio-discovery/templates/configmap.yaml manifests/charts/istiod-remote/templates/` this is needed for gateway. 
for the jwks one, I removed it.